test/metabase/lib/aggregation_test.cljc

```diff
@@ -244,6 +244,170 @@
+                  (lib/expression "budget?" (lib/< (lib/field (meta/id :venues :price)) 2))
+                  (lib/aggregate (lib/sum [:expression {:lib/uuid (str (random-uuid))} "double-price"])))
+        summable-cols [{:display-name "Latitude",
+                        :effective-type :type/Float,
+                        :semantic-type :type/Latitude,
+                        :lib/source :source/table-defaults}
+                       {:display-name "Longitude",
+                        :effective-type :type/Float,
+                        :semantic-type :type/Longitude,
+                        :lib/source :source/table-defaults}
+                       {:display-name "Price",
+                        :effective-type :type/Integer,
+                        :semantic-type :type/Category,
+                        :lib/source :source/table-defaults}
+                       {:display-name "double-price",
+                        :effective-type :type/Integer
+                        :lib/source :source/expressions}]
+        all-cols [{:display-name "ID"
+                   :effective-type :type/BigInteger
+                   :semantic-type :type/PK
+                   :lib/source :source/table-defaults}
+                  {:display-name "Name"
+                   :effective-type :type/Text
+                   :semantic-type :type/Name
+                   :lib/source :source/table-defaults}
+                  {:display-name "Category ID",
+                   :effective-type :type/Integer,
+                   :semantic-type :type/FK,
+                   :lib/source :source/table-defaults}
+                  {:display-name "Latitude",
+                   :effective-type :type/Float,
+                   :semantic-type :type/Latitude,
+                   :lib/source :source/table-defaults}
+                  {:display-name "Longitude",
+                   :effective-type :type/Float,
+                   :semantic-type :type/Longitude,
+                   :lib/source :source/table-defaults}
+                  {:display-name "Price",
+                   :effective-type :type/Integer,
+                   :semantic-type :type/Category,
+                   :lib/source :source/table-defaults}
+                  {:display-name "double-price"
+                   :effective-type :type/Integer
+                   :lib/source :source/expressions}
+                  {:display-name "budget?"
+                   :effective-type :type/Boolean
+                   :lib/source :source/expressions}
+                  {:display-name "ID",
+                   :effective-type :type/BigInteger,
+                   :semantic-type :type/PK,
+                   :lib/source :source/implicitly-joinable}
+                  {:display-name "Name",
+                   :effective-type :type/Text,
+                   :semantic-type :type/Name,
+                   :lib/source :source/implicitly-joinable}]
+        scope-cols all-cols
```

Comment:
For some unknown to me reason, the `scope` type, which is required for `min` and `max`, includes the `category` type which in turn includes booleans. This is unexpected, because we say that booleans are not ordered.

The current behavior (stats.metabase.com) is that min and max aggregations are offered for booleans but running the query fails with `function max(boolean) does not exist`.